tests/components/axis/test_diagnostics.py

```diff
@@ -38,6 +38,11 @@ async def test_entry_diagnostics(
             "unique_id": REDACTED,
             "disabled_by": None,
         },
+        "camera_sources": {
+            "Image": "http://1.2.3.4:80/axis-cgi/jpg/image.cgi",
+            "MJPEG": "http://1.2.3.4:80/axis-cgi/mjpg/video.cgi",
+            "Stream": "rtsp://user:pass@1.2.3.4/axis-media/media.amp?videocodec=h264",
```

Comment:
Shouldn't we redact the password in the URL?

Reply:
I think `generic` has a `redact_url`

Reply:
I'm replacing user and password in the generate source method. I don't want to use the redact method as that is the cause of issue I'm the referenced issue